src/Burger.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import {Footer} from './Footer.js'
+
+import {Lettuce,Bacon,Meat,Cheese} from './Ingredients.js'
+
+class Burger extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state={
+      Lettuce :0,
+      Bacon:0,
+      Cheese:0,
+      Meat:0
+    };
+  }
+
+  Add(i) {
+    const ingredients = {...this.state}
+    switch (i) {
+      case "Lettuce":
+        ingredients.Lettuce=ingredients.Lettuce +1
+        break;
+      case "Bacon":
+        ingredients.Bacon=ingredients.Bacon +1
+        break;
+      case "Cheese":
+          ingredients.Cheese=ingredients.Cheese +1
+        break;
+      case "Meat":
+        ingredients.Meat=ingredients.Meat +1
+        break;
+      }
+    this.setState({
+      Lettuce:ingredients.Lettuce,
+      Bacon:ingredients.Bacon,
+      Cheese:ingredients.Cheese,
+      Meat:ingredients.Meat
+    });
+    this.BurgerPrice();
+  }
+  Remove(i) {
+    const ingredients = {...this.state}
+    switch (i) {
+      case "Lettuce":
+        ingredients.Lettuce=ingredients.Lettuce -1
+        break;
+      case "Bacon":
+        ingredients.Bacon=ingredients.Bacon -1
+        break;
+      case "Cheese":
+          ingredients.Cheese=ingredients.Cheese -1
+        break;
+      case "Meat":
+        ingredients.Meat=ingredients.Meat -1
+        break;
+      }
+    this.setState({
+      Lettuce:ingredients.Lettuce,
+      Bacon:ingredients.Bacon,
+      Cheese:ingredients.Cheese,
+      Meat:ingredients.Meat
+    });
+
+    this.BurgerPrice();
+  }
+
+  MakeIngredient(key,value){
+    let ingredients=[]
+    switch (key) {
+      case "Lettuce":
+        ingredients=Array.from({length:value},(v,k)=>k+1).map(x => <Lettuce />)
+        break;
+      case "Bacon":
+        ingredients=Array.from({length:value},(v,k)=>k+1).map(x => <Bacon />)
+        break;
+      case "Cheese":
+        ingredients=Array.from({length:value},(v,k)=>k+1).map(x => <Cheese />)
+        break;
+      case "Meat":
+        ingredients=Array.from({length:value},(v,k)=>k+1).map(x => <Meat />)
+        break;
+      }
+
+    return ingredients
+  }
+
+  BurgerIndredients(){
+    return Object.entries(this.state).map(([key, value]) => this.MakeIngredient(key,value))
+  }
+
+  BurgerPrice(){
+    const BasicPrice=8
```

Comment:
Declare them as contants at the top of the class like 
```
const BASEPRICE=8
```

Reply:
Done